project-codeflare/codeflare-operator
# Updating make manifests target

# Issue link
Closes #364 

# What changes have been made
- Update to `manifests` make target allowing for division of crd files using yq.
- Addition of `install-yq` make target. 
- Update to crd/kustomization.yaml to include separate crds.
- Update to mcad/kustomizaton.yaml with latest version.
- Adding created CRD files.

# Verification steps
- Checkout this PR
- Run `make manifests` and ensure that it works as expected

## Checks
- Testing Strategy
